app/controllers/skills/new.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.ObjectController.extend({
+  needs: ['skills'],
+  // replace this with data from rails api call (get all groups)
+  groups: ["1407", "1409", "1410", "1412"],
```

Comment:
Hopefully we will be able to get all the groups from a rails api call (which will include instructors)